.github/workflows/test.yml

```diff
@@ -67,6 +67,15 @@ jobs:
       - name: Check that all features can compile
         run: cargo hack check --feature-powerset --depth 1
 
+  check-licenses:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Swatinem/rust-cache@v2
+      - name: Check that the 3rd-party license file is up to date
+        run: cargo install --git https://github.com/DataDog/rust-license-tool
```

Comment:
I assume this'll change when we have the first official release?

Reply:
What do you see as deficient with this approach? That is, change to what?

Reply:
Ah, I just imagined we might want to version the license tool and, potentially, even publish it to install from crates.io.

Reply:
I guess, eventually. We're not there yet.